docs/testplans/study-viewer-HEAL/study-viewer-HEAL.md

```diff
@@ -0,0 +1,96 @@
+# HEAL PROJECT Study Viewer: Tiered Access Support
+
+## Overview
+The Study Viewer should be able to display both open-access and controlled-access study level metadata. The user can 
```

Comment:
^ i think this sentence should say "The Study Viewer should be able to display open-access study level metadata."